### DISCUSSION TITLE
GitHub trending shows lot of empty categories

### DISCUSSION THREAD

**Discussion by yairlenga** [community/community#82755] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I am a big fun of using GitHub trending as a source for learning about new and emerging projects. I found it annoying that the drop down list (programming language) contain many entries without any projects (ABNP, , ABAP, …).

looks like the list is derived from list of possible languages. Anyone know who can change it so that it will list only languages with projects in them ?

**Comment by ghostinhershell** [comment]
Hey there @yairlenga 👋🏾 

Thanks so much for writing in. Can you clarify this feedback a bit? For example, do you mean that you're not seeing posts that are relevant to you or the programming languages that you're interested in within in the [Programming Help](https://github.com/orgs/community/discussions/categories/programming-help) community?

**Comment by yairlenga** [comment]
The drop down for languages contain many entries with no data. For users who would like to exlplore new projects it takes a lot of attempts to find setting that produce useful information. Try selecting “ABNO” - there is no projects for this category. Asking that the list will be filtered to items with projects.
